src/openms/source/FILTERING/DATAREDUCTION/MassTraceDetection.cpp

```diff
@@ -267,14 +276,48 @@ namespace OpenMS
       return;
     } // end of MassTraceDetection::run
 
+    double MassTraceDetection::find_offset_(Size peak_index_in_apices_vec, double mass_error_ppm_, const PeakMap& input_exp, const std::vector<Apex>& apices_vec)
```

Comment:
```suggestion
    double MassTraceDetection::find_offset_(Size peak_index_in_apices_vec, double mass_error_ppm_, const PeakMap& intput_map, const std::vector<Apex>& apices_vec)
```

input_exp may not be concise name.